controllers/containerjfr_controller.go

```diff
@@ -113,7 +116,13 @@ func (r *ContainerJFRReconciler) Reconcile(ctx context.Context, request ctrl.Req
 	// Check if this Recording is being deleted
 	if instance.GetDeletionTimestamp() != nil {
 		if controllerutil.ContainsFinalizer(instance, cjfrFinalizer) {
-			return r.deleteConsoleLinks(context.Background(), instance)
+			if r.IsOpenShift {
+				r.deleteConsoleLinks(context.Background(), instance)
```

Comment:
We should check for an error returned from deleteConsoleLinks here since we're no longer returning its return values.

Reply:
you're right, I made it return an error on purpose but never checked it!